src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx

```diff
@@ -80,6 +89,22 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
         }
     }, [appDetails])
 
+    const _getAndSetAppDetail = async () => {
+        try {
+            if (appDetails?.deploymentAppType === DeploymentAppType.argo_cd) {
+                const response = await getInstalledChartDetailWithResourceTree(
+                    +appDetails.installedAppId,
+                    +appDetails.environmentId,
+                )
+                IndexStore.publishAppDetails(response.result, AppType.DEVTRON_HELM_CHART)
+            }
+        } catch (e) {
```

Comment:
As we are handling errors via loading message, is there any requirement to show both loading message and toast together? Please ensure with product